charts/spinnaker/minimal-values.yaml

```diff
@@ -0,0 +1,22 @@
+
+halyard:
+  gitops:
+    enabled: true
+    secretName: opsmx-gitops-auth
+    repo:
+      type: git # git, bitbucket-stash or S3, please use another template for github/stash
+      baseUrlHostName: github.com    # Specify the Base FQDN of your repository without the protocol
+      organization: GIT_ORGANIZATION
+      repository: GIT_REPO_NAME # repo name for GitOps Halyard (Sample Reference: https://github.com/OpsMx/sample-gitops-repo.git).
+      halConfigBranch: main # Branch under which halyard config is present under above repository
+      halConfigPath: / #relative path from repository root folder
+      dynamicAccRepository: GIT_REPO_NAME # Please provide the repo name of the GitOps Dynamic Accounts Directory. Can be same as Hal repo.
+      dynAccntConfigPath: / #relative path from repository root folder
+      username: GIT_USERNAME  # Username to authenticate with git/stash repo
+      token: GIT_TOKEN
+      halConfigBranch: main      
```

Comment:
@keerthibingi There are multipe entries in the values file .Can you please remove it ??
halConfigBranch

Reply:
removed @saitejaopsmx